### DISCUSSION TITLE
understrapacademy is no more?

### DISCUSSION THREAD

**Discussion by samrueby** [holger1411/understrap#2225] [Q&A] [answered]
DNS does not resolve for https://www.understrapacademy.com. Is this site gone?

**Comment by howarddc** [comment, marked as answer]
We are retiring the Understrap Academy domain and brand, but if you own the Instant Understrap course already, you can sign in here to access it: http://robhoward.thinkific.com/

**Reply by samrueby** [reply]
Okay, thank you.
